test/flows/maternity_paternity_pay_leave_flow_test.rb

```diff
@@ -0,0 +1,552 @@
+      add_responses two_carers: "yes",
+                    due_date: "2016-1-1",
+                    employment_status_of_mother: "employee"
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      should "have a next node of mother_started_working_before_continuity_start_date for any response" do
+        assert_next_node :mother_started_working_before_continuity_start_date, for_response: "employee"
+      end
+
+      should "have a next node of mother_worked_at_least_26_weeks for any response" do
+        add_responses employment_status_of_mother: "self-employed"
+        assert_next_node :mother_worked_at_least_26_weeks, for_response: "employee"
+      end
+    end
+  end
+
+  context "question: mother_started_working_before_continuity_start_date" do
+    setup do
+      testing_node :mother_started_working_before_continuity_start_date
+      add_responses two_carers: "yes",
+                    due_date: "2016-1-1",
+                    employment_status_of_mother: "employee",
+                    employment_status_of_partner: "employee"
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      should "have a next node of mother_still_working_on_continuity_end_date for any response" do
+        assert_next_node :mother_still_working_on_continuity_end_date, for_response: "yes"
+      end
+    end
+  end
+
+  context "question: mother_still_working_on_continuity_end_date" do
+    setup do
+      testing_node :mother_still_working_on_continuity_end_date
+      add_responses two_carers: "yes",
+                    due_date: "2016-1-1",
+                    employment_status_of_mother: "employee",
+                    employment_status_of_partner: "employee",
+                    mother_started_working_before_continuity_start_date: "yes"
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      should "have a next node of mother_earned_more_than_lower_earnings_limit for any response" do
+        assert_next_node :mother_earned_more_than_lower_earnings_limit, for_response: "yes"
+      end
+    end
+  end
+
+  context "question: mother_earned_more_than_lower_earnings_limit" do
+    setup do
+      testing_node :mother_earned_more_than_lower_earnings_limit
+      add_responses two_carers: "yes",
+                    due_date: "2016-1-1",
+                    employment_status_of_mother: "employee",
+                    employment_status_of_partner: "employee",
+                    mother_started_working_before_continuity_start_date: "yes",
+                    mother_still_working_on_continuity_end_date: "yes"
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      should "have a next node of partner_started_working_before_continuity_start_date for a 'yes' response" do
```

Comment:
I think this needs something in the description to say what makes it different from the other "yes" response test.
```suggestion
      should "have a next node of partner_started_working_before_continuity_start_date for a 'yes' response if partner is an employee" do
```